Modules/Scripted/EditorLib/LabelSelectDialog.py

```diff
@@ -0,0 +1,94 @@
+
+import qt
+import slicer
+
+#-----------------------------------------------------------------------------
+def _map_property(objfunc, name):
```

Comment:
Thanks for your comments.

It seems the function is not specific to QObject, it allows to create a property associated with the attribute of an other object.

Here is an updated description:

```
  """Creates a python :class:`property` associated with a QObject
  property ``name``. ``objfunc`` is a function that takes a ``self`` as
  parameter and returns the QObject to consider.
  """
```